From 8fca529ceddb1b37d60948c37da6f2e0bf55de4d Mon Sep 17 00:00:00 2001
From: Chris Morgan <macromorgan@hotmail.com>
Date: Wed, 30 Oct 2024 21:06:34 -0500
Subject: [PATCH 16/17] arm64: dts: rockchip: Enable DMA for uart1 on RGxx3

Enable DMA by adding the proper dma-names for uart1 on the RGxx3.
This allows the UART controller to then use DMA.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
index 01988019743a..6cc86d21ffd1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-anbernic-rgxx3.dtsi
@@ -801,6 +801,7 @@ &tsadc {
 };
 
 &uart1 {
+	dma-names = "tx", "rx";
 	pinctrl-0 = <&uart1m1_xfer &uart1m1_ctsn &uart1m1_rtsn>;
 	pinctrl-names = "default";
 	uart-has-rtscts;
-- 
2.51.0

